doc/manual/parallel-computing.rst

```diff
@@ -451,36 +550,33 @@ preemptively. This means context switches only occur at well-defined
 points: in this case, when :func:`remotecall_fetch` is called.
 
 
-Channels
```

Comment:
Tasks are explained here - http://docs.julialang.org/en/release-0.4/manual/control-flow/#man-tasks . A `Channel` can be used to message/communicate between tasks in the same process.

In order to send a message to a task on a different process we can use a `RemoteChannel`.